jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -537,6 +537,18 @@ private String checkClassName(String className) {
 		final int length, 
 		ProtectionDomain protectionDomain) 
 		throws java.lang.ClassFormatError 
+{
+	return defineClassInternal(className, classRep, offset, length, protectionDomain, false);
```

Comment:
Extreme nitpick - code above uses a variable for the boolean, this code does not.

Reply:
I'd personally prefer an inline value with a comment to minimize bytecode size when running in the interpreter.

Reply:
Comments added at both call sites.